systemd/varnish.sysusers

```diff
@@ -0,0 +1,2 @@
+u varnishlog -:varnish "Varnish Cache Proxy"
+g varnish -
```

Comment:
Aren't we supposed to add the following credentials?

- varnish:varnish
- vcache:varnish
- varnishlog:varnish

Reply:
interesting, I didn't question the official packages, but it looks like they were indeed incomplete. Right now, all the files in `/var/lib/varnish` are owned by `root`. But once `varnish.sysusers` files is tweaked like this:

```
u varnishlog -:varnish "Varnish Cache Proxy"
u vcache -:varnish "Varnish Cache Proxy"
u varnish -:varnish "Varnish Cache Proxy"
g varnish -
```

Then we start seeing `vcache` and `varnish`, I guess `vanrishd` just silently fails if it can't find the users?

The files shown above works, but it's a case of "monkey see, monkey do", so if you have a more precise idea of what it should be, let me know

Reply:
Look in the `varnishd` manual for `-j unix`: those credentials will be used to drop privileges when available (except the `varnishlog` user that we use in the systemd `varnishncsa` service).

Reply:
yes, the "when available" surprised me as we are usually more deliberate. I've added the commit to the PR